xcode/Subconscious/Shared/Components/Common/Profile/UserProfileHeaderView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  UserProfileHeaderView.swift
+//  Subconscious (iOS)
+//
+//  Created by Ben Follington on 29/3/2023.
+//
+
+import SwiftUI
+
+struct UserProfileHeaderView: View {
```

Comment:
This is the header:

<img width="384" alt="image" src="https://user-images.githubusercontent.com/5009316/229039273-f03e7678-5546-40c2-9863-ab1482554e4a.png">

Reply:
Updated to use the outline button in all cases 👍